proteinworkshop/datasets/base.py

```diff
@@ -331,6 +341,11 @@ def download(self):
 
         Downloaded files are stored in ``self.raw_dir``.
         """
+        if self.format == "ent":  # Skip downloads from ASTRAL
+            logger.warning(
+                "Downloads in .ent format are assumed to be from ASTRAL. These data should have already been downloaded"
+            )
+            return
```

Comment:
Wouldn't we also need to skip calling `process()` in the `ProteinDataset` class in this context, since otherwise it would get called and it would try to process undownloaded ASTRAL files, for example? I am seeing precisely this issue when I try running this code when only updating the `download()` method.